corehq/apps/reports/standard/export.py

```diff
@@ -474,3 +527,45 @@ def selected_exports_data(self):
                 'exporttype': 'form',
             } for export in self.exports
         }
+
+
+class NewCaseExportReport(CaseExportReport):
+    base_template = 'reports/standard/export_download.html'
+    report_template_path = 'reports/partials/download_case_export.html'
+    name = ugettext_noop('Download Cases')
+    section_name = ugettext_noop('Export Data')
+    slug = 'case_export'
+
+    dispatcher = DataDownloadInterfaceDispatcher
+
+    @property
+    def template_context(self):
+        context = super(NewCaseExportReport, self).template_context
+        # TODO - seems redundant, cleanup at some point
+        context.update({
+            'export': self.exports[0],
+            # 'exports': self.exports,
```

Comment:
Will remove commented out code if we don't need to support bulk case downloads - checking on this